packages/feedback/src/types/index.ts

```diff
@@ -299,6 +303,14 @@ export interface FeedbackTheme {
    * Border styles for form inputs when focused
    */
   inputOutlineFocus: string;
+  /**
+   * Border radius for dialog
+   */
+  formBorderRadius: string;
```

Comment:
hmm, is `form` clear here as a prefix (from a user pov) - meaning is it clear what this border radius is referring to?

Reply:
we use `formTitle`, and refer to Feedback form in the docs, so this should be clear